### DISCUSSION TITLE
Welcome to openapi-react-query-codegen Discussions!

### DISCUSSION THREAD

**Discussion by 7nohe** [7nohe/openapi-react-query-codegen#79] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DawidWraga** [comment]
Hi just to let you know this package was extremely helpful for a previous project I was working on. Just wanted to say thanks :)

**Comment by trabati** [comment]
I agree, extremely helpful. Thank you, keep up the good work!

**Comment by nasersafaee** [comment]
is there an official documentation for migrating from 1.6.2 version to 2.0.0 version?
